utils/tests/utils.go

```diff
@@ -567,3 +568,35 @@ func CleanUpOldItems(baseItemNames []string, getActualItems func() ([]string, er
 		}
 	}
 }
+
+// Set HomeDir to desired location.
+// Caller is responsible to set the old home location back.
+func SetJfrogHome(newHome string) (oldHome string, err error) {
```

Comment:
I noticed that in ```TestUnitTests``` this function is used, without a defer clause that reverts back to the old home dir. Let's add it.